appendix.Rmd

```diff
@@ -77,4 +83,15 @@ knitr::write_bib(pkgs, file="packages.bib")
 knitr::kable(pkgs_df)
 ```
 
+These can be installed as follows:
+
+```{r, eval=FALSE}
+u = "https://git.io/vX8aa"
```

Comment:
No, just that the previous way people don't see a way of getting a data frame of required packages on their computer. If you don't know that the `install_github()` command also installs all required packages, it may seem confusing. Also, it means that people without devtools will be able to install the packages.